docs/internal-documentation/distrib-testing/Debian-Preparation.md

```diff
@@ -40,3 +39,13 @@
    ```sh
    exit
    ```
+
+
+## Preparing to install the repositories
+
+1. Follow the steps (1-7) of the [release
+process](https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/maintenance/ReleaseProcess.md)
+to prepare the tarballs in a separate machine
```

Comment:
I think we want to link to the ReleaseProcess document in the same branch as the document as one is reading:

```
1. Follow the steps (1-7) of the [release process] to prepare the tarballs in a separate machine

[release process]: ../maintenance/ReleaseProcess.md
```